doc/api/buffer.md

```diff
@@ -1340,6 +1340,10 @@ console.log(b.indexOf('b', null));
 console.log(b.indexOf('b', []));
 ```
 
+If `value` is an empty string or `Buffer` and `byteOffset` is less
```

Comment:
Yea, makes sense. I went with the former because `is empty` sounds a bid odd to me (like empty/undefined variables?), and `has a length of zero` doesn’t make sense for integer input (and users can figure that out, but, you know), so I chose the imho explicit version.